src/main/java/de/dafuqs/spectrum/items/food/SedativesItem.java

```diff
@@ -30,7 +32,10 @@ public ItemStack finishUsing(ItemStack stack, World world, LivingEntity user) {
 				}
 				
 			}
-			
+			if(FabricLoader.getInstance().isModLoaded("neepmeat"))
```

Comment:
this one should check if the integration pack is active ("neepmeat installed and integration pack not disabled")